src/System.Net.Requests/tests/FileWebRequestTest.cs

```diff
@@ -80,7 +80,7 @@ public void GetRequestResponseAfterAbort_Throws()
         [Fact]
         public void NotImplementedMembers_Throws()
         {
-            WebRequest request = WebRequest.Create("file://anything");
+            WebRequest request = WebRequest.Create("file:///anything");
```

Comment:
These paths previously worked on both Windows and Unix without the third slash.  Is that changing?